kong/plugins/rate-limiting/schema.lua

```diff
@@ -44,9 +48,9 @@ return {
           }, },
           { policy = {
               type = "string",
-              default = "cluster",
+              default = "local",
```

Comment:
Thinking out loud: can this be classified as a breaking change?

To be clear, I'm not advocating against it but would like to clarify the compat promise that Kong 1.0 brings with it.

Reply:
Agreed, the default should change only on DB-less.

Reply:
Thanks for pointing that, @hbagdi. The default policy is `"local"` only for DB-less now.